chain/jsonrpc/tests/rpc_query.rs

```diff
@@ -142,24 +142,51 @@ fn test_query_account() {
             .query(RpcQueryRequest {
                 block_id: None,
                 request: QueryRequest::ViewAccount { account_id: "test".to_string() },
+                finality: Finality::None,
             })
             .await
             .unwrap();
         let query_response_2 = client
             .query(RpcQueryRequest {
                 block_id: Some(BlockId::Height(0)),
                 request: QueryRequest::ViewAccount { account_id: "test".to_string() },
+                finality: Finality::None,
             })
             .await
             .unwrap();
         let query_response_3 = client
             .query(RpcQueryRequest {
                 block_id: Some(BlockId::Hash(block_hash)),
                 request: QueryRequest::ViewAccount { account_id: "test".to_string() },
+                finality: Finality::None,
             })
             .await
             .unwrap();
-        for query_response in [query_response_1, query_response_2, query_response_3].into_iter() {
+        let query_response_4 = client
+            .query(RpcQueryRequest {
+                block_id: Some(BlockId::Hash(block_hash)),
+                request: QueryRequest::ViewAccount { account_id: "test".to_string() },
+                finality: Finality::DoomSlug,
+            })
+            .await
+            .unwrap();
+        let query_response_5 = client
+            .query(RpcQueryRequest {
```

Comment:
How would this query look if it was made via a terminal? Would one have to spell out "Finality::NFG"?

Also, we should not leak internal terms NFG and Doomslug into the public API. It should be something along the lines of:

"last" for None
"final" for NFG

For doomslug we need something clever. May be "near" or "fast-final". Or may be just "default" :)
Will start a poll in slack in #development.

Reply:
API expects just a enum variant name by default, so "NFG" | "DoomSlug" | "None". It is a great point of having better public-facing naming. Thanks for catching this!